tests/golden/common/common.bats

```diff
@@ -43,7 +43,6 @@ EOF
 
```

Comment:
Imo it would be better if we split fixing this test case in another commit.

Reply:
I could, but it would mean on one stage our test's won't  pass. Is it acceptable? And this changes directly connected to code changes